contracts/proxies/SafeProxyFactory.sol

```diff
@@ -40,7 +40,11 @@ contract SafeProxyFactory {
             /// @solidity memory-safe-assembly
             assembly {
                 if eq(call(gas(), proxy, 0, add(initializer, 0x20), mload(initializer), 0, 0), 0) {
-                    revert(0, 0)
+                    let returnPtr := mload(0x40)
```

Comment:
this is a change in behaviour that I don't think we want `revert(0,0)` is always memory-safe.

Reply:
Will revert in next commit 👍🏾